VEHICLE/IsPedExclusiveDriverOfVehicle.md

```diff
@@ -1,17 +1,18 @@
 ---
 ns: VEHICLE
-aliases: ["0xB09D25E77C33EB3F"]
+aliases: ["0xB09D25E77C33EB3F", "_IS_PED_EXCLUSIVE_DRIVER_OF_VEHICLE"]
 ---
-## _IS_PED_EXCLUSIVE_DRIVER_OF_VEHICLE
+## IS_PED_EXCLUSIVE_DRIVER_OF_VEHICLE
 
 ```c
-// 0xB09D25E77C33EB3F
-BOOL _IS_PED_EXCLUSIVE_DRIVER_OF_VEHICLE(Ped ped, Vehicle vehicle, int outIndex);
+// 0xB09D25E77C33EB3F 0xFF62D324
```

Comment:
I'm not sure adding joaat hashes is supported, this might break existing resources. 

Where did you get this hash from?

Reply:
The joaat hash was taken from [VEHICLE/SetVehicleExclusiveDriver.md](https://github.com/citizenfx/natives/pull/625/files/f18980e435cce3f26e18a51f9f792c14e0aad3bd#diff-b48a74e0d1a1f86a6838ac21ddf2e4ed0835746e2b1735a7d619b440520d6eae).

As you can see, the previous joaat hashes didn't match. 

If you hash "IS_PED_EXCLUSIVE_DRIVER_OF_VEHICLE" you will get 0xFF62D324.